vacasa/connect/connect.py

```diff
@@ -2082,6 +2082,27 @@ def get_unit_bulk(self, ids: str):
         return self._post(url, json={'data': {'attributes': payload}}, headers=headers).json()
 
 
+    def get_tickets(self, params: dict = None):
+        """Retrieve a list of tickets
+        Yields:
+            An iterator of tickets. Each ticket is a dict.
+        """
+        url = f"{self.endpoint}/v1/tickets"
+        headers = self._headers()
+
+        return self._iterate_pages(url, headers, params)
+
+    def get_ticket_by_id(self, ticket_id):
+        """ Get a single ticket by ID """
+        if not ticket_id:
+            return None
+        url = f"{self.endpoint}/v1/tickets/{ticket_id}"
+        try:
+            return self._get(url, headers=self._headers()).json()['data']
+        except (HTTPError, KeyError):
+            return None
```

Comment:
Looking at the other comparable methods in this file, I don't see this kind of error handling. Probably best to be consistent. Any reason to treat it differently for tickets?

Reply:
Actually, I think I think I modeled it after the only method with that kind of error handling lol. I'll refactor!